workspace.json

```diff
@@ -1741,7 +1750,33 @@
             "cwd": "apps/application-system/api"
           }
         },
-        "docker-express": {}
+        "docker-express": {},
+        "dev/init": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "commands": [
+              "yarn dev-services application-system-api",
+              "yarn nx run application-system-api:migrate",
+              "yarn nx schemas/build-openapi application-system-api",
+              "yarn nx schemas/openapi-generator api-domains-application",
+              "yarn nx schemas/build-graphql-schema api"
+            ],
+            "parallel": false
+          }
+        },
+        "dev": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "commands": [
+              "yarn start api",
+              "yarn start application-system-api",
+              "yarn start application-system-form",
+              "kubectl port-forward svc/socat-soffia 8443:443 -n socat",
+              "kubectl port-forward svc/socat-xroad 8081:80 -n socat"
```

Comment:
This should use our `scripts/run-*-proxy.sh` scripts.

Reply:
Which proxy script does `svc/socat-soffia` correspond to?

Reply:
Added the script here 🙄